### ISSUE TITLE
비전 모델 만들기

### ISSUE THREAD

**Issue by soojlee0106** [DevthanAI/MentalHabitBack#2]
Pytorch 혹은 Tensorflow

**Comment by soojlee0106** [comment]
https://towardsdatascience.com/bounding-box-prediction-from-scratch-using-pytorch-a8525da51ddc

**Comment by soojlee0106** [comment]
https://medium.com/analytics-vidhya/implement-your-own-mask-rcnn-model-65c994a0175d

**Comment by soojlee0106** [comment]
https://universe.roboflow.com/models/object-detection
